rgw/v2/tests/s3_swift/test_versioning_with_objects.py

```diff
@@ -600,6 +600,31 @@ def test_exec(config, ssh_con):
                             bucket.name,
                             VERSIONING_STATUS["SUSPENDED"],
                         )
+                        if config.test_versioning_archive:
+                            arc_site_ip = utils.get_rgw_ip_arc()
+                            arc_site_ssh_con = utils.connect_remote(arc_site_ip)
+                            bucket_stats_op = utils.exec_shell_cmd(
+                                f"radosgw-admin bucket stats --bucket {bucket_name_to_create}"
+                            )
+                            bucket_stats_json_doc = json.loads(bucket_stats_op)
+                            bucket_id = bucket_stats_json_doc["id"]
+                            cmd_arc_metadata = f"radosgw-admin metadata get --metadata-key bucket.instance:{bucket_name_to_create}:{bucket_id}"
+                            stdin, stdout, stderr = arc_site_ssh_con.exec_command(
+                                cmd_arc_metadata
+                            )
+                            metadata_bucket_get = json.loads(stdout.read().decode())
+                            log.info(
+                                f"Metadata for the bucket {bucket_name_to_create} is {metadata_bucket_get}"
+                            )
+                            log.info(
+                                "Check the value of flag, it should be 2 for version enabled buckets"
+                            )
+                            if metadata_bucket_get["data"]["bucket_info"]["flags"] != 2:
```

Comment:
Small query,
@viduship , what would be the value for versioning disabled bucket?

Reply:
0 or 6